src/coreclr/jit/emitxarch.cpp

```diff
@@ -16497,6 +16545,7 @@ size_t emitter::emitOutputInstr(insGroup* ig, instrDesc* id, BYTE** dp)
             else
             {
                 code    = AddSimdPrefixIfNeeded(id, code, size);
+                code    = AddEvexbBitIfNeeded(id, code);
```

Comment:
Should this just be handled in `AddSimdPrefixIfNeeded` since its part of the EVEX prefix and only needed if that is itself being added?